docs/data/material/components/material-icons/SearchIcons.js

```diff
@@ -110,6 +111,12 @@ const StyledIcon = styled('span')(({ theme }) => ({
   },
 }));
 
+const PlaceholderIcon = styled('div')({
```

Comment:
This could be slow, to consider the need for one class name shared with the SVG placeholders.

Reply:
Ah yeah actually, this sounds great. How about:

1. We kill `const StyledSvgIcon = styled(SvgIcon)(({ theme }) => ({`
2. We kill `PlaceholderIcon`
3. We replace `StyledSvgIcon` by one style at the root
4. We apply this style